amgut/lib/data_access/ag_data_access.py

```diff
@@ -625,12 +628,18 @@ def AGGetBarcodeMetadataAnimal(self, barcode):
         return return_res
 
     def getAnimalParticipants(self, ag_login_id):
-        results = self._sql.execute_proc_return_cursor(
-            'ag_get_animal_participants', [ag_login_id])
-
-        return_res = [row[0] for row in results]
-        results.close()
-        return return_res
+        sql = """SELECT participant_name FROM ag_animal_survey
```

Comment:
It's only used in `get_menu_items`, and the animal survey handler, so as
long as it works with both of those were good? Recommend an appropriate doc
string though

On Mon, Jul 27, 2015 at 4:10 PM, adamrp <notifications@github.com> wrote:

> In amgut/lib/data_access/ag_data_access.py
> <https://github.com/biocore/american-gut-web/pull/432#discussion_r35593610>
> :
>
> > @@ -625,12 +628,18 @@ def AGGetBarcodeMetadataAnimal(self, barcode):
> >          return return_res
> >
> >      def getAnimalParticipants(self, ag_login_id):
> > -        results = self._sql.execute_proc_return_cursor(
> > -            'ag_get_animal_participants', [ag_login_id])
> > -
> > -        return_res = [row[0] for row in results]
> > -        results.close()
> > -        return return_res
> > +        sql = """SELECT participant_name FROM ag_animal_survey
>
> Yeah, I think that makes sense to do. @wasade <https://github.com/wasade>,
> thoughts? The main reason I'm concerned is that the function
> getAnimalParticipants might be used in places where it is assumed that a
> set of operations can be applied equally to each item in the output, which
> might not be the case if there is more than one source for the data.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/american-gut-web/pull/432/files#r35593610>.
>